tsconfig.base.json

```diff
@@ -1,20 +1,20 @@
 {
   "compilerOptions": {
+    "target": "es2015",
```

Comment:
Are we missing the schema definition/reference on the top? 
` "$schema": "http://json.schemastore.org/tsconfig",`

Reply:
I'm not sure, is this normally needed? It seems at least for vscode it is aware of the schema and I get completions, but I can add that if there are issues